OrbitGl/App.cpp

```diff
@@ -1027,49 +997,58 @@ void OrbitApp::LoadModules() {
   if (m_ModulesToLoad.size() > 0) {
     if (Capture::IsRemote()) {
       LoadRemoteModules();
-    } else {
+      return;
+    }
+#ifdef _WIN32
+    std::shared_ptr<Module> module = m_ModulesToLoad.front();
```

Comment:
When you put a std::move in here you safe a lot of mutex-synchronisation time.

Reply:
done